english/sonic-screwdriver/sonic-screwdriver.md

```diff
@@ -6,4 +6,6 @@ When you open the door back in Londo you realize that you left your trusty sidek
 
 Do you:
 
-[Go back and find the impossible girl?](../impossible-girl/impossible.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.